src/logger.rs

```diff
@@ -10,6 +10,18 @@ use std::{
 
 static HAS_ERROR: AtomicBool = AtomicBool::new(false);
 
+macro_rules! suggest {
+    () => {
+        eprintln!()
+    };
+    ($($arg:tt)*) => {
+        eprint!("{}", ::crossterm::style::Stylize::blue("HINT "));
+        eprintln!($($arg)*);
+    };
+}
```

Comment:
Should we make our message headings lowercase and boldface?

Reply:
Sure, that sounds reasonable to me! Maybe I will save this for a different PR (when fixing #96, where we need to deal with a few other things).